packages/opentelemetry-metrics/src/index.ts

```diff
@@ -25,4 +25,6 @@ export * from './export/aggregators';
 export * from './export/Batcher';
 export * from './export/ConsoleMetricExporter';
 export * from './export/types';
+export * from './export/Batcher';
```

Comment:
Any reason this export is repeated here?

Reply:
Nope :)